src/cogito/package.d

```diff
@@ -36,6 +36,30 @@ private Result runOnFile(string file)
     return typeof(return)(visitor.source);
 }
 
+/**
+ * Map a filename to an array of files.
+ *
+ * Params:
+ *     filename = File path.
+ *
+ * Returns: Array of filenames.
+ *   If the filename points to a folder all d-files are returned,
+ *   If the filename points to a file an array with only that file is returned.
+ */
+string[] toFiles(string filename)
+{
+    import std.file;
+    if (filename.isFile)
+    {
+        return [filename];
+    }
+    if (filename.isDir)
+    {
+        return filename.dirEntries("*.d", SpanMode.breadth).map!("a.name").array;
```

Comment:
`dirEntries` is as bad as the `find` command. With `find src -name "*.d"` I get
```
src/cogito/list.d
src/cogito/package.d
src/cogito/meter.d
src/cogito/visitor.d
src/cogito/arguments.d
src/main.d
```

That's not exactly user friendly: I would look for `src/cogito/arguments.d` at the top.

Reply:
:) ... whatever the filesystem provides ... but i think we can sort the files anyways ...